include/domain/containers/creation_handler.h

```diff
@@ -20,8 +20,8 @@ namespace domain::containers
 {
     struct creation_configuration
     {
-        std::string container_base_folder;
-        std::string image_base_folder;
+        std::string containers_folder;
```

Comment:
I'm more used to "directory" than "folder". The latter resembles more Windows environments. Just my 2cents of rant (:

Reply:
😆 it's pretty valid.
 I'm not very familiar with the conventions for windows unfortunately 😆 .